src/Rules/Test.hs

```diff
@@ -15,47 +15,70 @@ import System.Environment
 -- TODO: clean up after testing
 testRules :: Rules ()
 testRules = do
+    root <- buildRootRules
+
+    -- | Using program shipped with testsuite to generate ghcconfig file.
+    root -/- ghcConfigProgPath ~> do
+        ghc             <- builderPath $ Ghc CompileHs Stage0
+        command [] ghc [ghcConfigHsPath, "-o" , root -/- ghcConfigProgPath]
```

Comment:
Could you switch to using `cmd` instead of `command` as in the rest of Hadrian?

Reply:
Resolved